cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/field_display_filtering.spec.js

```diff
@@ -0,0 +1,178 @@
+      const matches = $url.match(timeRegex);
+      cy.wrap(matches).should('have.length.above', 0); // There should be a fromTime and endTime
+
+      const fromTime = matches[1];
+      const toTime = matches[2];
+
+      if (
+        (isNowTime(fromTime) && isRelativeTime(toTime)) ||
+        (isRelativeTime(fromTime) && isNowTime(toTime))
+      ) {
+        cy.setSearchAbsoluteDateRangeWithSearchDatePickerButton(START_TIME, END_TIME);
+      } else {
+        cy.setSearchAbsoluteDateRangeWithStartEndDateButtons(START_TIME, END_TIME);
+      }
+    });
+  }
+
+  switch (datasetType) {
+    case 'index_pattern':
+      if (language !== 'OpenSearch SQL') {
+        setAbsoluteDateRangeUsingDatePickerButtonOrStartEndTimeButtons();
+      }
+      break;
+  }
+}
+
+function verifyTableFieldFilterActions(datasetType, language, shouldExist) {
+  selectDataset(datasetType, language);
+  setDateRange(datasetType, language);
+
+  cy.getElementByTestId('docTable').get('tbody tr').should('have.length.above', 3); // To ensure it waits until a full table is loaded into the DOM, instead of a bug where table only has 1 hit.
+
+  dataExplorer.verifyDocTableRowFilterForAndOutButton(0, shouldExist);
+
+  if (shouldExist) {
+    dataExplorer.verifyDocTableFilterAction(0, 'filterForValue', '10,000', '1', true);
+    dataExplorer.verifyDocTableFilterAction(0, 'filterOutValue', '10,000', '9,999', false);
+  }
+}
+
+function verifyExpandedTableFilterActions(datasetType, language, isEnabled) {
+  selectDataset(datasetType, language);
+  setDateRange(datasetType, language);
+
+  cy.getElementByTestId('docTable').get('tbody tr').should('have.length.above', 3); // To ensure it waits until a full table is loaded into the DOM, instead of a bug where table only has 1 hit.
+  dataExplorer.toggleDocTableRow(0);
+  dataExplorer.verifyDocTableFirstExpandedFieldFirstRowFilterForFilterOutExistsFilterButtons(
+    isEnabled
+  );
+  dataExplorer.verifyDocTableFirstExpandedFieldFirstRowToggleColumnButtonHasIntendedBehavior();
+
+  if (isEnabled) {
+    dataExplorer.verifyDocTableFirstExpandedFieldFirstRowFilterForButtonFiltersCorrectField(
+      0,
+      0,
+      '10,000',
+      '1'
+    );
+    dataExplorer.verifyDocTableFirstExpandedFieldFirstRowFilterOutButtonFiltersCorrectField(
+      0,
+      0,
+      '10,000',
+      '9,999'
+    );
+    dataExplorer.verifyDocTableFirstExpandedFieldFirstRowExistsFilterButtonFiltersCorrectField(
+      0,
+      0,
+      '10,000',
+      '10,000'
+    );
+  }
+}
+
+describe('filter for value spec', () => {
+  before(() => {
+    // Load test data
+    cy.setupTestData(
+      SECONDARY_ENGINE.url,
+      ['cypress/fixtures/query_enhancements/data-logs-1/data_logs_small_time_1.mapping.json'],
+      ['cypress/fixtures/query_enhancements/data-logs-1/data_logs_small_time_1.data.ndjson']
+    );
+    // Add data source
+    cy.addDataSource({
+      name: `${DATASOURCE_NAME}`,
+      url: 'http://opensearch-node:9200',
```

Comment:
Replace  this with   `url: `${SECONDARY_ENGINE.url}`` . 

USe `url: 'http://opensearch-node:9200'` only in local while you are testing you changes in the docker setup

Reply:
Fixed, thank you.